main/src/ca/uwaterloo/flix/language/errors/TypeError.scala

```diff
@@ -274,4 +274,84 @@ object TypeError {
     }
   }
 
+  /**
+    * Error indicating that the types of two instances overlap.
+    *
+    * @param loc1 the location of the first instance.
+    * @param loc2 the location of the second instance.
+    */
+  case class OverlappingInstance(loc1: SourceLocation, loc2: SourceLocation) extends TypeError {
```

Comment:
Is this error message duplicated? i.e. it should exist for both loc1, loc2, and loc2, loc1. 
(For VScode)